js/main.js

```diff
@@ -21,4 +21,68 @@ form.addEventListener('submit', function (event) {
   data.entries.push(userInput);
   img.setAttribute('src', 'images/placeholder-image-square.jpg');
   form.reset();
+  $formContainer.className = 'container ' + 'hidden';
+  $entryContainer.className = 'container';
+  data.view = 'previous-entries';
```

Comment:
What is `previous-entries`? You should only have an `entry-form` view and a `entries` view.